kdaglas/nomaCakes
# #169204251 Setup the django nomacakes project

### What does this PR do?
This PR setup the django project

### Description of Task to be completed?
- create the django project
- add the database configuration
- create the app files
- add migrations

### How should this be manually tested?
- Fetch this branch by running the command `git fetch origin ch-setup-169204251` in the terminal and check into the branch `git checkout ch-setup-169204251`
- create a virtual environment to hold your packages using `virtualenv venv`
- Start the virtual environment `source venv/bin/activate`
- make migrations so that the fields required for the data are initiated by typing this command: `python manage.py migrate`
- Start the application by running this command: `python manage.py runserver`
- Go to postman and run the port `http://127.0.0.1:8000/`

### Story
[#169204251](https://www.pivotaltracker.com/story/show/169204251)
